pkgs/development/python-modules/gtimelog/default.nix

```diff
@@ -31,6 +47,26 @@ buildPythonPackage rec {
 
   pythonImportsCheck = [ "gtimelog" ];
 
+  # found this fix at Matrix
+  # see https://matrix.to/#/!KqkRjyTEzAGRiZFBYT:nixos.org/$reRwjXiP3c4q9-z3wnnnBF9Q5KYGi9w3DxeQLhEJhnI?via=nixos.org&via=matrix.org&via=tchncs.de
+  # resolves runtime error: Typelib files for Gtk-3.0 are not available.
+  makeWrapperArgs = [
+    "--set GI_TYPELIB_PATH ${lib.makeSearchPathOutput "lib"
+      "lib/girepository-1.0" ([
+        gtk3
+        libsoup
+        libsecret
+        pango
+        harfbuzz
+        gdk-pixbuf
+        atk
+      ])}"
+    "--set GIO_MODULE_DIR ${lib.makeSearchPathOutput "out"
+      "lib/gio/modules" ([
+        glib-networking
+      ])}"
+  ];
+
   preFixup = ''
     wrapProgram $out/bin/gtimelog \
```

Comment:
The proper way is to add `wrapGappsHook` to `nativeBuildInputs`. Ideally with this in `preFixup` to avoid double wrapping: https://nixos.org/manual/nixpkgs/stable/#ssec-gnome-common-issues-double-wrapped